packages-exp/analytics-exp/src/constants.ts

```diff
@@ -43,7 +43,7 @@ export const enum GtagCommand {
  *
  * @public
  */
-export const enum EventName {
+export enum EventName {
```

Comment:
enum is not treeshakeable. It's helpful for developers to see what events are available, but we also define function overloads for all of them, which should provide the same convenience. Since it's kinda of a big object and will likely grow in size and we have event name overloads for logEvent(), I don't think we should export it.

I also notice that it is only used in tests internal, so perhaps we can remove it completely or move it to a test only file?

Reply:
Ok I guess they can rely on the function overloads. Removed. Removed from test files too since the tests should probably call the functions as users would - with string literals.